src/nr_openai_observability/langchain_callback.py

```diff
@@ -7,6 +7,9 @@
 from nr_openai_observability import monitor
 import newrelic.agent
 
+CompletionEventName = "LlmCompletion"
+ChainEventName = "LlmChain"
+ToolEventName = "LlmTool"
```

Comment:
You might be able to add these to / reference these from the `consts.py` file

Reply:
Will do. This is part of the fun of merging all the branches back together :)